tensorflow/lite/experimental/micro/tools/make/Makefile

```diff
@@ -225,6 +225,10 @@ CXX := $(CC_PREFIX)${TARGET_TOOLCHAIN_PREFIX}${CXX_TOOL}
 CC := $(CC_PREFIX)${TARGET_TOOLCHAIN_PREFIX}${CC_TOOL}
 AR := $(CC_PREFIX)${TARGET_TOOLCHAIN_PREFIX}${AR_TOOL}
 
+# Default target must appear before any target, 
```

Comment:
We looked at this change - any reason why you needed to move this rule up?

Reply:
To let `make -f tensorflow/lite/experimental/micro/tools/make/Makefile TARGET=riscv32_mcu` (call the default make target), works the same as `make -f tensorflow/lite/experimental/micro/tools/make/Makefile TARGET=riscv32_mcu _all_`.
The upmost make target is the default target for Make, if without this patch, the default target looks unexpected.

Move this rule up to let `all` be the upmost Make target.

Described in this issue.
https://github.com/tensorflow/tensorflow/issues/33677